frontend/src/metabase/visualizations/visualizations/SmartScalar/compute.unit.spec.js

```diff
@@ -125,11 +125,11 @@ describe("SmartScalar > compute", () => {
         description:
           "should correctly display percent decrease over missing row",
```

Comment:
The description for the test case doesn’t make sense anymore after your change below.

This “missing row” here is intended to be October.  If the “latest” data is for November, and the most recent data is for September, then the comparison would read “Sep” instead of “previous month”.  Does that make sense?

Reply:
Yes, thanks for catching that!